src/components/Game/Slots/Slots.js

```diff
@@ -7,44 +7,30 @@ import { WORD_LENGTH } from '../../../constants/constants';
 import { generateID } from '../../../helpers';
 
 
-const Slots = ({ lettersInSlots, clicked }) => {
-    const slotsNodes = [];
-    let slotIndex = 0;
-
-    while (slotIndex < WORD_LENGTH) {
-        const letter = lettersInSlots && lettersInSlots[slotIndex];
-        const letterIndex = slotIndex;
-
-        slotsNodes.push(
-            <li
-                key={generateID()}
-                className={classes.Slot}
-            >
-                {
-                    letter
-                        ? (
-                            <span
-                                className={classes.SlotLetter}
-                                onClick={() => clicked(letter.id, letterIndex)}
-                            >
-                                {letter.value}
-                            </span>
-                        )
-                        : null
-                }
-            </li>,
-        );
-        slotIndex += 1;
-    }
-
-    return (
-        <section className={classes.SlotsSection}>
-            <ul className={classes.Slots}>
-                {slotsNodes}
-            </ul>
-        </section>
-    );
-};
+const Slots = ({ lettersInSlots, clicked }) => (
+    <section className={classes.SlotsSection}>
+        <ul className={classes.Slots}>
+
+            {[...Array(WORD_LENGTH).keys()].map((i) => {
+                const letter = lettersInSlots[i] || null;
```

Comment:
I believe `|| null` is not needed

even nicer way `.map(i => lettersInSlots[i]).map(letter => (…`

Reply:
👍